lib/potluck.rb

```diff
@@ -0,0 +1,37 @@
+require 'pry'
+require './lib/dish'
+
+class Potluck
+  attr_reader :dishes, :add_dish
+
+  def initialize(date)
+    @date = date
+    @dishes = []
+    @category = {appetizer: "", entre: "", dessert: ""}
+  end
+
+  def date
+    @date
+  end
+
+  def add_dish(name)
+    @dishes << Dish.new(name, category)
```

Comment:
You don't want to call Dish.new here. The argument passed in is a dish object, so you just have to shovel it into your dishes array.